packages/runtime/container-runtime/src/summary/summarizer.ts

```diff
@@ -151,7 +151,7 @@ export class Summarizer extends EventEmitter implements ISummarizer {
 		// This will result in "summarizerClientDisconnected" stop reason recorded in telemetry,
 		// unless stop() was called earlier
 		this.dispose();
-		(this.runtime.disposeFn ?? this.runtime.closeFn)();
+		this.runtime.disposeFn();
```

Comment:
Should this just live in the `dispose` function?

Reply:
The runtime's `disposeFn` will actually call `Container.dispose` instead of just `ContainerRuntime.dispose`